src/models/Expense.model.js

```diff
@@ -1,9 +1,50 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+
+    spentAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+
+    amount: {
+      type: DataTypes.INTEGER,
```

Comment:
The `amount` field is defined as an `INTEGER`. Depending on the use case, it might be more appropriate to use `DataTypes.DECIMAL` or `DataTypes.FLOAT` if the amount can have decimal values. Consider changing this if your application requires handling of fractional amounts.